docs/ref/request-response.txt

```diff
@@ -936,6 +936,11 @@ JsonResponse objects
    are allowed). If ``safe`` is ``True`` and a non-``dict`` object is passed as
    the first argument, a :exc:`TypeError` will be raised.
 
+   .. versionchanged:: 1.9
```

Comment:
This annotation should be "self-contained" as described in https://docs.djangoproject.com/en/1.8/internals/contributing/writing-documentation/#documenting-new-features.